### PR TITLE
In order to prevent Github Actions from running on branches ending with "from-video"

### PR DESCRIPTION
**Pull request by aatwi**
Update the "on" section in the 3 yml files to include: 
1. Push on master 
2. PullRequests to master

### CODE REVIEW

**Comment by AntoineMx** [comment]
I'm not sure we want that actually. We did detect issues on development branches and I feel it would be best to detect them as soon as possible. PRs can contain a lot of changes and it might be difficult to pinpoint an issue.
Since we mainly work on Windows and have Linux pipelines we are bound to detect undefined behaviors with them (and those are the hardest to fix)

**Review by AntoineMx — CHANGES_REQUESTED** [review]
I'm not sure we want that actually. We did detect issues on development branches and I feel it would be best to detect them as soon as possible. PRs can contain a lot of changes and it might be difficult to pinpoint an issue.
Since we mainly work on Windows and have Linux pipelines we are bound to detect undefined behaviors with them (and those are the hardest to fix)

**Comment by aatwi** [comment]
@AntoineMx I was debating this argument in my head :)
However, with what we have now, the argument is that the build will fail even if the Master branch is compiling (that is what happened with Philippe yesterday because he intentionally submitted uncompilable code to a demo-branch for demo purposes)

**Comment by AntoineMx** [comment]
> 
> 
> @AntoineMx I was debating this argument in my head :)
> However, with what we have now, the argument is that the build will fail even if the Master branch is compiling (that is what happened with Philippe yesterday because he intentionally submitted uncompilable code to a demo-branch for demo purposes)

Actually, nothing prevents someone from creating a branch removing the .yml file from it from the beginning. That way the pipelines are launched on all branches unless the ones for which the author deliberately and explicitly disabled them.

File: .github/workflows/cmake.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 name: CMake
 
-on: [push]
+on: 
+  push:
+    branches-ignore:
+      - mikado-method-from-video
```
Comment (philou) [inline .github/workflows/cmake.yml:6]: Why didn't you put the regexp here?

**Review by philou — CHANGES_REQUESTED** [review]

File: .github/workflows/cmake.yml (hunk 1 above)
Comment (aatwi) [reply .github/workflows/cmake.yml]: It was not properly saved on my machine. 
I submitted a fix for that

**Review by AntoineMx — APPROVED** [review]

**Review by mengdaming — APPROVED** [review]

**Pull request merged**